controllers/restic.go

```diff
@@ -116,13 +116,22 @@ func (r *DataMoverBackupReconciler) buildResticSecret(secret *corev1.Secret, dmb
 		case key == ResticPassword:
 			ResticPasswordValue = val
 		case key == ResticRepository:
+
+			// check for trailing '/' in user-created Restic repo
+			// if included, remove it
+			lastCharInRepo := val[len(val)-1:]
```

Comment:
@eemcmullan Thanks for PR. For this logic, I would suggest something like `val = strings.TrimRight(val, "/")` 
WDYT ?